scripts/game/prove-features-gate.ps1

```
@@ -62,12 +62,44 @@ function Write-GateResult([hashtable]$Result) {
     $Result | ConvertTo-Json -Depth 10 | Set-Content -LiteralPath $gateResultPath -Encoding utf8
 }
 
-function Test-GameAvailable {
+$script:GameProcessBaseName = 'Diplomacy is Not an Option'
+$script:GameExecutableFileName = 'Diplomacy is Not an Option.exe'
+
+function Stop-StrayGameLaunchProcesses {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Stop-StrayGameLaunchProcesses' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)